apps/feedback/views/content/start.md

```diff
@@ -1,7 +1,6 @@
 {{#renderField}}feedback{{/renderField}}
 
 {{#renderField}}improvements{{/renderField}}
-<p>{{#t}}pages.start.paragraph-1-hint{{/t}}</p>
 
```

Comment:
With the updated HOF packages we now have a more accessible way to impose a maximum character count on a text area. The maximum character count is now part of the text area mixin, rather than a static message on the page, and counts down the number of words remaining as the user types.

<img width="673" alt="Screenshot 2021-02-26 at 11 11 44" src="https://user-images.githubusercontent.com/61828376/109293234-6c3c1f00-7823-11eb-9a6d-8fc1505ba349.png">

Reply:
Oooh nice